DEVELOPING.md

```diff
@@ -0,0 +1,706 @@
+# Developing Emissary
+
+Table of Contents
+=================
+* [Requirements](#requirements)
+* [IDE integration](#ide-integration)
+* [Coding Standards](#coding-standards)
+* [Internals](#internals)
```

Comment:
Reading through this, this felt out of place here. I personally like the old organization with the Internals first and then the "how to" with IDE/standards/commands/etc.

Reply:
I moved the section back to the top and renamed it overview.